clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1014,6 +1040,11 @@ private long getLastConsumedMessageTimestamp(int partition) {
   }
 
   protected boolean shouldNewLeaderSwitchToRemoteConsumption(PartitionConsumptionState partitionConsumptionState) {
+    LOGGER.info(
+        "PCS: {}, isConsumingFromRemoteVersionTopic: {}, isLeaderConsumingRemoteRealTimeTopic: {}",
+        partitionConsumptionState,
+        isConsumingFromRemoteVersionTopic(partitionConsumptionState),
+        isLeaderConsumingRemoteRealTimeTopic(partitionConsumptionState));
```

Comment:
Was this for debugging during implementation? If no, let's use the task name prefix in the log.